CA_DataUploaderLib/ServerUploader.cs

```diff
@@ -792,6 +807,7 @@ public Signing(string loopName)
             }
 
             public byte[] GetPublicKey() => _rsaWriter.ExportCspBlob(false);
+            public byte[] GetPrivateKey() => _rsaWriter.ExportCspBlob(true);

```

Comment:
do we need after the approach change at the end? (if removing please remove the callers)

Reply:
removed